kernel/src/main/java/org/kframework/compile/ConvertDataStructureToLookup.java

```diff
@@ -121,7 +121,6 @@ public static Set<KLabel> filteredMapConstructors(Module m) {
     return stream(m.productions())
         .filter(p -> p.att().contains(Att.ASSOC()) && p.att().contains(Att.FILTER_ELEMENT()))
         .map(p -> p.klabel().get())
-        .distinct()
         .collect(Collectors.toSet());
```

Comment:
Isn't there a `.toSet()` variant to get rid of this extra line?

Reply:
Weirdly, no, despite the existence of `.toList()`!